packages/renderer/src/lib/troubleshooting/TroubleshootingPageStore.svelte

```diff
@@ -18,18 +22,18 @@ async function fetch(): Promise<void> {
   }
 }
 
-let openDetails = false;
+let openDetails = $state(false);
 </script>
 
 <div class="flex flex-col bg-[var(--pd-invert-content-card-bg)] p-2 items-center rounded-sm w-full" role="listitem" aria-label={eventStoreInfo.name}>
-  <div><svelte:component this={eventStoreInfo.iconComponent} size="20" /></div>
```

Comment:
Not related to this PR but this feel a bit hacky to store some icon component in a store?